tests/openadapt/test_scrub.py

```diff
@@ -62,7 +148,7 @@ def test_scrub_image() -> None:
 
     # Assert ~1.5% mask pixels compared to total pixels.
     assert (
-        round((mask_pixels / total_pixels), 3) == 0.015
+        round((mask_pixels / total_pixels), 3) == 0.022
```

Comment:
Updated the values acc. to he changes in the latest update in the presidio image-redactor library